cli/pcluster/utils.py

```diff
@@ -580,3 +582,13 @@ def retry(func, func_args, attempts=1, wait=0):
 
             LOGGER.debug("{0}, retrying in {1} seconds..".format(e, wait))
             time.sleep(wait)
+
+
+def retry_on_boto3_throttling(func, wait=5, *args, **kwargs):
+    while True:
+        try:
+            return func(*args, **kwargs)
+        except ClientError as e:
+            if e.response["Error"]["Code"] != "Throttling":
+                raise
+            time.sleep(wait)
```

Comment:
Let's make this dynamically extend like an exponential backoff or the like

Reply:
do you see a real value in having a exponential backoff here? In the end the retry happens every 5 seconds and I wouldn't extend it more than that.
Also this call is not the real responsible of the throttling but rather the call from the compute nodes where I agree an exponential backoff is the more appropriate choice.